scrapper/city_scrapper.py

```diff
@@ -54,23 +54,20 @@ def get_city_details(self, city_details_html):
         try:
             city_details_soup = BeautifulSoup(city_details_html, "html.parser")
             text = city_details_soup.find(class_="text")
-
-            self._logger.debug(f"City details - <div class=\"text\">...<div>: {text}")
+            # self._logger.debug(f"City details - <div class=\"text\">...<div>: {text}")
 
             if not text:
                 return
-
+            self._logger.info('Fetching tabs info...')
             city = text.h1.text if text.h1 else "-"
             country = text.h2.text if text.h2 else "-"
 
             tabs = city_details_soup.find("div", class_="tabs").find("div", class_="ul").find_all("h2", class_="li")
-            self._logger.debug(f"City details - Tabs: {tabs}")
+            # self._logger.debug(f"City details - Tabs: {tabs}")
             tabs_information = {TabScrapper.get_name(tab): self._get_tab_information(tab, city_details_soup)
                                 for tab in tabs if TabScrapper.is_valid(tab)}
-            self._logger.debug(f"City details - Tabs Information: {tabs_information}")
-
-            self._logger.info(f"All the information about {city}, {country} was fetched!")
-
+            # self._logger.debug(f"City details - Tabs Information: {tabs_information}")
+            # self._logger.info(f"All the information about {city}, {country} was fetched!")
             # TODO get the rank
```

Comment:
Does someone want to get the rank? It's in the 1st tab (scores), inside the 1st row/